l10n/de_DE/LC_MESSAGES/funding.po

```diff
@@ -91,6 +91,14 @@ msgstr "Ausbezahlter Betrag"
 msgid "The sum of the amounts of accepted drawdowns."
 msgstr "Die Summe der Beträge akzeptierter Mittelabrufe."
 
+#: Civi/Funding/Api4/Action/FundingCase/GetFieldsAction.php managed/SavedSearchFundingCases.mgd.php
+msgid "Review Progress"
+msgstr "Prüffortschritt"
+
+#: Civi/Funding/Api4/Action/FundingCase/GetFieldsAction.php
+msgid "The progress of application review."
+msgstr "Zeigt die Anzahl der geprüften Anträge."
```

Comment:
I think `Anzahl` is the wrong word. Literally translated would it be something like `Der Fortschritt der Antragsprüfung.` Thinking about it I'm wondering if the text (English as well as German) should include that it's in %?

Reply:
To add Percent is a good idea. Will change the german translation as well.